### PR TITLE
Added breadcrumbs navigation on specific recipe pages

### PR DESCRIPTION
**Pull request by asia-t**
Resolves: https://github.com/ertrzyiks/yummy/issues/27

### CODE REVIEW

File: src/components/breadcrumbs/breadcrumbs.module.sass [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  margin: 0
+
+.breadcrumbs li
+  color: #606060
+  text-transform: uppercase
+  display: inline-block
+  margin: 0
+
+  &:not(:first-child):before
+    content: "/"
```
Comment (ertrzyiks) [inline src/components/breadcrumbs/breadcrumbs.module.sass]: lets use `font-size: 19px`

File: src/components/breadcrumbs/breadcrumbs.module.sass [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.breadcrumbs
+  list-style: none
+  margin: 0
+
+.breadcrumbs li
```
Comment (ertrzyiks) [inline src/components/breadcrumbs/breadcrumbs.module.sass]: There is no strong reason to use tag name, what about `.breadcrumb-item`?

File: src/templates/post-page.js [hunk 3]
```diff
@@ -15,12 +16,14 @@ export default function Template({data}) {
     return <Link to={'/tag/' + slug} key={tag} className={[buttonClass, postStyles.post_tag].join(' ')}>{tag}</Link>
   })
 
+  const categorySlug = recipe.slug.substring(1).split('/')[0]
```
Comment (ertrzyiks) [inline src/templates/post-page.js]: To be improved in #55

**Comment by asia-t** [comment]
The changes are in.

**Pull request merged**